src/MeetingsSDKAdapter.test.js

```diff
@@ -1000,45 +1000,20 @@ describe('Meetings SDK Adapter', () => {
     });
   });
 
-  describe('switchSpeakerControl()', () => {
-    test('returns the display data of a meeting control in a proper shape', (done) => {
-      meetingSDKAdapter.switchSpeakerControl(meetingID)
-        .pipe(take(1)).subscribe((dataDisplay) => {
-          expect(dataDisplay).toMatchObject({
-            ID: 'switch-speaker',
-            tooltip: 'Speaker Devices',
-            options: null,
-            selected: null,
-          });
-          done();
-        });
-    });
-
-    test('throws errors if sdk meeting object is not defined', (done) => {
-      meetingSDKAdapter.fetchMeeting = jest.fn();
-
-      meetingSDKAdapter.switchSpeakerControl(meetingID).subscribe(
-        () => {},
-        (error) => {
-          expect(error.message).toBe('Could not find meeting with ID "meetingID" to add switch speaker control');
-          done();
-        },
-      );
-    });
-  });
-
   describe('switchSpeaker()', () => {
-    beforeEach(() => {
-      meetingSDKAdapter.meetings[meetingID] = {
-        speakerID: null,
-      };
-    });
-
-    test('emits the switch speaker events with speakerID', async () => {
+    test('sets the speaker that was chosen by the user', async () => {
+      meetingSDKAdapter.meetings[meetingID].speakerID = null;
       await meetingSDKAdapter.switchSpeaker(meetingID, 'speakerID');
 
-      expect(mockSDKMeeting.emit).toHaveBeenCalledWith('adapter:speaker:switch', {
-        speakerID: 'speakerID',
+      expect(mockSDKMeeting.emit).toHaveBeenCalledTimes(1);
```

Comment:
let's refactor this (in a different subtask) in a function to be used like this:
```js
expect(mockSDKMeeting).toHaveEmittedObjectMatching('adapter:meeting:updated', {speakerID: 'speakerID'});
```

How to implement a custom matcher: https://jestjs.io/docs/expect#custom-matchers-api